sbws/core/generate.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, fp, bw, nick, rtts, last_time):
         rtts = [round(r * 1000) for r in rtts]
         self.rtt = round(median(rtts))
         self.time = unixts_to_isodt_str(last_time)
+        self.ed25519 = ed25519
 
     def __str__(self):
```

Comment:
The point of getting the ed25519 identity is to include it on the line, right? So if it is not None, we should include it here, right?

Reply:
yes, the point it's to print it in the bw line.
i think i put a comment somewhere that i was not adding that until #160 was merged, cause #160 was rewriting all that code.
i should rebase this to master again and print it in the bw line